homeassistant/components/hassio/handler.py

```diff
@@ -125,7 +127,6 @@ def get_discovery_message(self, uuid):
         options = {
             'ssl': CONF_SSL_CERTIFICATE in http_config,
             'port': port,
-            'password': http_config.get(CONF_API_PASSWORD),
```

Comment:
CC @pvizeli , we will no longer send the API password to Hass.io. This should be fine as we have switched to tokens.

Reply:
Is fine for me